### PR TITLE
fix: readme link

### PR DESCRIPTION
**Pull request by AlexD10S**
Fix README after an issue with the link.

Tested the fix locally https://github.com/AlexD10S/pop-cli

### CODE REVIEW

**Comment by weezy20** [comment]
Not seeing the `Pop` icon on the README. Neither on your fork either.

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Pop CLI
 
-<img src=".icons/logo.jpeg"></img>
+<img src="/.icons/logo.jpeg"></img>
```
Comment (weezy20) [inline README.md:3]: This doesn't appear to be working correctly. I can still not see the banner.

**Review by weezy20 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (AlexD10S) [reply README.md:3]: This make the logo appears in pop-cli/crate/README.md

**Comment by AlexD10S** [comment]
Sorry I missed a commit and got it wrong. README.md in `.github` folder removed.

**Review by al3mart — APPROVED** [review]

**Pull request merged**